### ISSUE TITLE
Suggestions for new features?

### ISSUE THREAD

**Issue by LividJava** [EhsanKia/CatalogScanner#14]
I have no idea if you want these in two separate issues but i'll pool them together for now.

Having the scanner support Nook Mile achievements, Some people like tracking their achievements and ACNH Planner has a section for it to be tracked already!

The ability to find previous scans on the site, Maybe by caching or doing a lookup by user?? this isn't a big deal as you can just search for them on twitter.

**Comment by EhsanKia** [comment]
Both good ideas. Achievement scanning is a bit tricky and I have attempted it in the past. It's also doubly tricky to find an export format that other sites like ACNH Planner will be able to parse. I might give it a try this weekend.

The other one should be fairly easily, I might be able to do it quickly tonight.

**Comment by LividJava** [comment]
For parsing something like `{ACHIEVEMENTNAME}:{0-6}` 0 being the achievement is present and each number there after representing which level you've unlocked?

You could try talking to Carolin in her discord and asking about it? she's very responsive

**Comment by LividJava** [comment]
@linsvensson thoughts??

**Comment by EhsanKia** [comment]
Yes, I do have contacts with most of the devs that integrate with the bot. I'll settle on a reasonable format and then let them all now so they can add support for it. Just gotta make sure whatever name or id I use is something common which everyone can convert to/from. Note that the bot supports all languages too so using some id may be better than the english name.

The other feature is finished, not the prettiest but you can go to https://nook.lol/user/<twitter_handle> to get the top 20 most recent scans.
